pkg/aws/awscluster/mutate_awscluster.go

```diff
@@ -431,6 +443,42 @@ func (m *Mutator) MutateRegion(awsCluster infrastructurev1alpha2.AWSCluster) ([]
 	return result, nil
 }
 
+//MutateAnnotationNodeTerminateUnhealthy migrate NodeTerminateUnhealthy annotations from alpha to stable in case it is configured.
+// this migration code can be removed once all AWS clusters are on release 15.0.0 or newer
+func (m *Mutator) MutateAnnotationNodeTerminateUnhealthy(awsCluster infrastructurev1alpha2.AWSCluster) ([]mutator.PatchOperation, error) {
+	var result []mutator.PatchOperation
+
+	release, err := semver.New(key.Release(&awsCluster))
+	if err != nil {
+		return nil, microerror.Maskf(notAllowedError, fmt.Sprintf("AWSCluster release has invalid value '%s'.", key.Release(&awsCluster)))
+	}
+
+	// new annotation is available from release >= 15.x.x
+	release15 := semver.MustParse("14.99.99")
```

Comment:
I had do this trick as when testing `15.0.0` was greater than `15.0.0-calvix` and it kinda breaks my testing, this should be safe